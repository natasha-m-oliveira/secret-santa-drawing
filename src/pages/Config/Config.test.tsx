import { render } from '@testing-library/react';
import React from 'react';
import { RecoilRoot } from 'recoil';
import { Config } from './';

const mockNavigate = jest.fn();

jest.mock('react-router-dom', () => {
  return {
    useNavigate: () => mockNavigate
  }
})

describe('the config page', () => {
  test('should be rendered correctly', () => {
    const {container} = render(
      <RecoilRoot>
        <Config />
      </RecoilRoot>
    )

    expect(container).toMatchSnapshot();
  })
})