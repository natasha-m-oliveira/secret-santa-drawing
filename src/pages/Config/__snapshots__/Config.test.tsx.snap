// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the config page should be rendered correctly 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos come√ßar!
      </h2>
      <form>
        <div
          class="group"
        >
          <input
            placeholder="Insira os nomes dos participantes"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Adicionar
          </button>
        </div>
      </form>
      <ul />
      <footer
        class="footer"
      >
        <button
          disabled=""
        >
          Iniciar brincadeira
        </button>
        <img
          src="assets/img/bags.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
